### ISSUE TITLE
A11y_Visual Studio Code_Sidebar Options_Link Purpose: "Show link" in the side bar options is not descriptive to understand

### ISSUE THREAD

**Issue by v-kanve** [microsoft/vscode#122834]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code;#LinkPurpose;#JAWS;#ScreenReader;#WCAG2.4.4;;#Win32

### Environment Details:​
Application Name: Visual Studio Code
Application Version: 1.55.2 (user setup)
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome:87.04280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Screen Reader: JAWS
Standard Violated: WCAG2.4.4

### Repro Steps:​
1. Turn on JAWS
2. Open Visual Studio Code editor
3. Navigate to the debug side view bar option using "Ctrl+Shift+D" key
4. Tab till "Show" link 
5. Verify whether the link is descriptive enough to understand or not

### Actual Result:
"Show link" in the side bar options is not descriptive to understand

### Expected Result: ​
"Show link" in the side bar options should be descriptive to understand. It should provide any information related to the link.

### User Impact:
Screen Reader users will face difficulty while navigation if show link is not descriptive. They will not understand the purpose of the link.

### Attachment:
[A11y_Visual Studio Code Client [Benchmark]_Sidebar Options_Link Purpose Show link in the side bar options is not descriptive to understand - Copy - Copy.zip](https://github.com/microsoft/vscode/files/6413452/A11y_Visual.Studio.Code.Client.Benchmark._Sidebar.Options_Link.Purpose.Show.link.in.the.side.bar.options.is.not.descriptive.to.understand.-.Copy.-.Copy.zip)

**Comment by isidorn** [comment]
To verify make sure that the "Show" link in the debug welcome view is now more descriptive. It is "Show all automatic debug configurations" now

**Comment by v-kanve** [comment]
@isidorn Could you please provide the build version in which it is fixed.

**Comment by isidorn** [comment]
@v-kanve insiders build in 4 days will have this fix. So starting on May 7th
